test/functional/Java9andUp/build.xml

```diff
@@ -114,6 +114,32 @@
 						</javac>
 					</sequential>
 				</for>
+				<!-- compile unnamed module depending on module org.openj9test.modularity.dummy -->
+				<for list="unnamed" param="mod">
```

Comment:
Is there a reason to use for loop for `unnamed` and `testUnreflect`? Both loops only have one item. Are we going to add more modules?

Reply:
There is no immediate plan to add more modules though this is the preferred place to add future modularity tests.
I felt that `for loop` distinguishes modules being added from existing ones, and looks clearer about the dependency and order to build the modules.